src/components/big-interactive-pages/editor.tsx

```diff
@@ -151,6 +152,47 @@ export const saveGame = debounce(
 	}
 );
 
+export async function saveGame2(persistenceState: Signal<PersistenceState>) {
+	const attemptSaveGame = async () => {
+		try {
+			const game =
+				persistenceState.value.kind === "PERSISTED" &&
```

Comment:
I believe I have observed this same conditional many times in the codebase - should it be wrapped in a convenient method to reduce # of LOC?

Reply:
I don't think so. It's only 2 conditions, basically 1 line of code